tests/l2/qat/qatlib_build.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Could we change the commit log to:

tests_l2: Add qatlib based Intel QAT sample workload

qatlib is built from source in https://github.com/intel/qatlib
The qatlib version is specified by QATLIB_VERSION: 23.08.0
cpa_sample_code shipped with qatlib is used to verify the QAT provisioning on OCP 

Resource:
qat.intel.com/dc is used for QAT compression/decompression 
qat.intel.com/cy is used for QAT asymmetric/asymmetric cryptography  

serviceAccount: intel-qat uses the user defined SCC intel-qat-scc which enables the IPC_LOCK capability for Qat-lib based workload see https://github.com/intel/intel-technology-enabling-for-openshift/issues/122

Signed-off-by: veenadhari.bedida@intel.com

Reply:
updated

Reply:
can update commit message. do we plan to do this for every PR?

Reply:
Yes, please. :-)